i18n-util/jest.config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  coverageDirectory: './test-reports/coverage',
```

Comment:
Not at the moment. I had started a thread in [eng-prod ](https://okta.slack.com/archives/C7LQ4U8T0/p1607974630246000) about this and since only a handful of devs will end up working in this repo. Bacon integration does not seem to be an immediate requirement.

Reply:
Cool - I'd suggest we remove the `jest-junit` dependency and this reporter logic then, since it isn't going to be needed.